modulefiles/gfsutils_generic.lua

```diff
@@ -0,0 +1,22 @@
+help([[
```

Comment:
What is the purpose of this module file?

Reply:
It is a version-less replacement for gfsutils_common.lua that is used when building with a container. We can add in the version numbers, if you want, but leaving them out allows more flexibility to update the container without having to update module files in all repos. I understand that having specific version numbers is desirable for testing and reproducibility, but allowing flexibility in them for the container case improves portability and extensibility.

Reply:
I understand it improves portability and extensibility, but they come at the cost of reproducibility, reliability, and maintainability.  Is there a way to have minimum versions in lua modulefiles?  In cmake one can provide a minimum version.
The `gfsutils_common.lua` provides a mechanism for the user to provide a specific version.  See [here](https://github.com/NOAA-EMC/gfs-utils/blob/develop/modulefiles/gfsutils_common.lua).  If the container has a different version, it can set those version numbers and still be able to leverage `gfsutils_common.lua`.  I think if `default` is provided, it would be similar to this file.

Reply:
I'll switch it around so that the container just uses the gfs_common.lua module file along with environment variables to specify the versions available to the container.